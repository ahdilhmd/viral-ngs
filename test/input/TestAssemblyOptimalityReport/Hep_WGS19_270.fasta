>Hep_WGS19_270-1
CCTTGGATTTGTTCCACTTTATGGACACTCATTTCACGCTTTCTGTCTGCTTTTCTTCCA
GGGCTCTCCCCTTGCCCTAGGCTCTGGCCGTTGCGCCCGGCNNNNNNNNNNCCATGATTA
GCATGGAGCTGTAGGAGTCTAAATTGGGGACACAGATGTTGGGAACGTCACCTTGCAGTG
TTAACTTGGCTTTCATGAAGCTTTTTGATCTTCCACAAGAGGTAGGCTACGGGTGAAACC
TCTTAAGCTAATACTTCTATGAAGAGATGCTTTGGATAGGGTAACAGCGGCGGACATTGG
TGNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNACAAAACTTGGCCTTAAATGAGGTCCTGTGAGAGGGGACCTCGCCATTGAATGCTG
GACATTTCTTTGGGGCCTTATGTTTTGTTTGCCTCTGAGGTACTCAGGGGCATTTAGGTT
TTTCCTCATCAATAAATAATTATGAATATGNNCAGGCAAGGTATTTTCCAGACTGTTGGG
AGTGGCCTTGACCACATCCTGTCTTTGGCAGATGTGGAGGAGGAACAAATGATTCAGTCT
GTGGATCGTACCGCAGTTACTGGGGCTTCATATTTCACTTCTGTGGATCAATCTTCTGTT
CATACAGCTGAAGTTGGCTCACATCAACCTGAACCTTTGAAAACCTCTGTTGACAAACCA
GGCTCTAAGAGGACACAAGGAGAGAAATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNGTTTCATGAAGTTNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNTTGCTGTTCAGGGTCTGTTGAGGTATCACACTTATGCAAGATTTGGAATTGAGATA
CAAGTTCAGATCAATCCCACACCATTCCAGCAAGGTGGTTTGATATGTGCCATGGTGCCA
GGAGATCAGAGCTATGGATCTATAGCTTCTTTGACAGTTTATCCCCATGGTTTGTTGAAT
TGTAATATCAACAATGTGGTCAGAATTAAGGTTCCTTTTATTTATACAAGAGGAGCTTAT
CACTTTAAGGACCCTCAATATCCCGTTTGGGAGTTGACTATTAGAGTTTGGTCTGAGCTA
AACATTGGAACTGGTACCTCCGCTTATACATCACTGAATGTGCTGGCTAGATTTACTGAT
TTGGAACTTCATGGGCTAACACCCTTGTCTACACAGATGATGAGAAATGAATTTAGAGTC
AGTACAACAGAAAATGTAGTTAATTTGTCCAATTATGANNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNTTTTACAACTTGGACATCAATTCCCACTTTGGCTGCCCAGTTTCCATTTAATGCT
TCTGATTCAGTTGGGCAACAGATTAAGGTTATTCCAGTTGATCCATATTTCTTCCAAATG
ACTAATACAAATCCTGAACAAAAATGTATAACTGCATTGGCTTCAATATGCCAAATGTTT
TGTTTTTGGAGAGGAGACTTGGTTTTTGACTTCCAGGTTTTTCCTACAAAATATCATTCA
GGAAGATTATTATTTTGTTTTGTCCCTGGAAATGAATTGATTGATGTTTCCCACATAACA
CTGAAACAAGCCACTACTGCGCCTTGTGCTGTGATGGATATTACTGGTGTACAGTCAACT
TTGAGATTTCGTGTTCCTTGGATTTCNNNNNNNCCTTANNNNNNNNNNNNNNNNNNNNNA
TCGTCACATCAGAAAGGAGAGTATACTGCCATAGGAAAGTTGATTGTTTACTGTTACAAC
AGACTGACTTCTCCCTCTAATGTGGCTTCCCATGTTAGAGTCAATGTATATCTCTCAGCT
ATTAATTTGGAATGTTTTGCTCCACTTTATCATGCCATGGATGTCACAACTCAGGTTGGA
GATGATTCAGGAGGCTTTTCTACTACTGTTTCAACAAAACAGAATGTTCCAGATCNNNNN
NNNNNTATCACAACAGTCAAGGATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNCATTTCTTGTGTACATTTACATTTAATTCT
AATAACAAAGAGTACACTTTTCCCATCACTTTGTCATCTACTTCTAATCCTCCTCATGGA
CTGCCTTCAACTCTGAGATGGNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNCTCTGGAT
CTGACAATAATTATAACTGGGGCTACTGATGNNNNNNNNNNNNNNNNGTTNNNNNNNNNN
NNNNNNNNNNNAGACACCCCATGGGTTGAGAAGGAGTCTGCTCTTTCCATTGATTACAAG
ACAGCTCTTGGTGCTGTTAGGTTTAATANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNCTAGAGCA
CCTTTGAACACCAATGCTATGATGTCATCAGAAACAATGATGGATAGAATTGCTCTTGGT
GATCTTGAATCCTCAGTTGATGATCCTCGATCTNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTGCCTCCTCCCCGTAAAATTAAAGGTGTG
TTTTCACAAGCAAAAATCTCATTGTTCTATACAGAAGATCATGAAATTATGAAATTTTCC
TGGAAAGGAATTACTGCTGACACTAGAGCTCTGAGGAGATTTGGTTTTTCATTGGCTGCT
GGTAGAAGTGTGTGGACATTGGAAATGGATGCTGGAGTTCTGACTGGTAGGTTGGTGAGG
GTCAATGATGAAAAATGGACAGAAATCAAAGATGACAAAATAGTTTCTTTGGTGGAGAAA
TTTACTAGTAATAAACACTGGTCCAAAGTTAATTTTCCTCATGGAATGTTGGATTTGGAN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTCTTG
TTGCACTGGCTGAACCCCAAAAAGATAAACTTGGCAGATAGAATGTTGGGTCTGTCAGGG
ATACAGGAAATAAAAGAAAAAGGGGTGGGATTGNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNTTAAAAGNNNNNNNNNGCTGTATGTNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTCTCAAATTGGCTT
AGAGACATTTGTTCAGGTATAACAATTTTTAAAAGTTTCAAGGATGNGATTTATTGGCTT
TACACAAAAATTAGAGAATACTATGATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNGCAATTGAGGAGGCTGATAATTTTTGT
GTTCTTCAGATTCAGGATGTTGAAAAATTTGAACAGTACCAAAAAGGGGTAGATTTAATT
CAAAAATTGAGAACTGTTCATTCAATGGCACAAGTTGATCCAGGTTTAACTGTTCATTTA
GCCCCTCTTAGAGACTGTATTGCTAGAGTTCATCAAAAGTTAAAAAATTTGGGTTCAATT
AACCAGGCAATGGTGACAAGATGTGAGCCAGTCGTCTGCTACCTCTATGGAAAGAGGGGT
GGTGGAAAAAGTCTGACTTCAATTGCTTTGGCCACAAAAATTTGTAAACATTATGGTGTT
GAGCCTGAGAAAAACATATACACAAAACCTGTTGCATCTGATTATTGGGATGGATATNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
CCTAAAACGGTCTATGTTAAGGAAGCCATAGACCGTAGACTCCATTTTAAAGTTGAGGTG
AAACCTGCCTCATTCTTCAAGAATCCTCATAATGATATGTTGAATGTTAATCTAGCAAAA
ACTAATGATGCTATTAAAGACATGTCATGTGTTGATCTTGTTATGGATAATCACAATGTT
TCACTGTCTGAATTACTTAGTTCACTGGTGATGACAGTTGAAATTAGAAAACAAAATATG
TCAGAATTTATGGAACTTTGGTCACAGGGATTATCAGATGATGACAATGATAGTGCAGTT
GCTGAATTTTTCCAGTCTTTTCCCTCTGGNNNNNNATCTGGTTCAAGGTTATCACAGTTT
TTCCAATCAGTTACTAACCACAAGTGGGTTGCAGTTGGGGCTGCTGTGGGAGTACTTGGT
GTTTTGGTTGGAGGTTGGTATGTCTACAAGCATTTTACAAAGAAGCAAGAAGAATCCATT
CCAAGTGAAGGAGTGTATCATGGTGTGACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNGTTAATGGGACTCCAATGTTAATTTCAGAGGGACCT
CTCAAAATGGAGGNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTCATCATCACATTGATAAAAACATGATT
AATTTTCCTGCAGTAATGCNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNGTGATGCTG
TCAAAGTATTCTNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNATAAAATTTTGGGTAAGAGTCCTCTAGTTGATGACTTTCTGGATATTGAAATG
GCTATAACAGGAGCTCCTGGAATAGATGCAATCAACATGGATTCTTCCNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
TGTTCAGATTTGGATGTTGTTTTCACCACTTGTCCCAAAGATGAATTAAGACCTCTAGAC
AAAGTATTGGAATCCAAAACTAGGGCTATTGATTCTTGTCCTCTTGACTATACAATTTTG
TGTAGAATGTACTGGGGTCCAGCCATTAGTTATTTCCATTTAAATCCTGGTTTTCATACT
GGTGTTGCAATTNGAATTGATCCTGATAGACAGTGGGACCAACTTTTTAAAACAATGATT
AGATTTGGAGATGTTGGTCTAGATTTGGATTTTTCCGCTTTTGATGCTAGTTTGAGTCCA
TTCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNGTATGTGGGTCAATGCCTTCTGGGTCTCCTTGTACAGCTTTGTTGAATTCAATTATA
AACAATGTTAATCTGTACTATGTCTTTTCCAAGATTTTCAAAAAATCTCCAGTTTTCTTT
TGTGATGCTGTGAGAATTCTTTGTTATGGAGATGATGTTCTTATTGTCTTCTCCAGACAG
GTTCAAATTGACAATCTGGATTCTATTGGACAAAAAATTGTTGATGAGTTTAAGAAATTG
GGAANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNACCAGTTTCTGAGTTA
ACATTTTTAAAAAGATCTTTTAATTTGGTTGAAGATAGAATCAGACCTGCTATTGCTGAG
AAAACCATATGGTCATTAGTTGCATGGCAAAGGAGCAATGCTGAATTTGAACAGAATTTA
GAGAATGCCCAGTGGTTTGCTTTTATGCATGGATATGAATTTTACCAACAATTTTATCAT
TTTGTTCAGTCCTGCTTGGAGAAAGAGATGATAGAATACAGA
