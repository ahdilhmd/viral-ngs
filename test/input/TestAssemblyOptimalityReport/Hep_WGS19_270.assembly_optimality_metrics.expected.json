{
    "10_cleaned": {
        "kmers": 1673041,
        "stage_num": 0,
        "taxon_kmers": 6776
    },
    "20_taxfilt": {
        "kmers": 7333,
        "stage_num": 1,
        "taxon_kmers": 6776,
        "taxon_kmers_lost_since_10_cleaned": 0
    },
    "30_contigs": {
        "kmers": 5917,
        "stage_num": 2,
        "taxon_kmers": 5851,
        "taxon_kmers_lost_since_20_taxfilt": 925
    },
    "40_intermediate_scaffold": {
        "kmers": 5930,
        "stage_num": 3,
        "taxon_kmers": 5899,
        "taxon_kmers_lost_since_30_contigs": 3
    },
    "50_scaffolded_imputed": {
        "kmers": 7421,
        "stage_num": 4,
        "taxon_kmers": 7371,
        "taxon_kmers_lost_since_40_intermediate_scaffold": 0
    },
    "60_refine1": {
        "kmers": 5623,
        "stage_num": 5,
        "taxon_kmers": 5598,
        "taxon_kmers_lost_since_50_scaffolded_imputed": 1795
    },
    "90_final": {
        "kmers": 3866,
        "stage_num": 6,
        "taxon_kmers": 3841,
        "taxon_kmers_lost_since_60_refine1": 1757
    },
    "taxon_kmers": 324246
}
